backend/src/Response/Order/OrderCancelResponse.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Response\Order;
+
+class OrderCancelResponse
+{
+    public int $id;
+
+    public string $state;
+
+    public string $statusError;
```

Comment:
Also, when we do not set a value for statusError (when we can cancel an order), then this would cause an error online.